opencog/moses/main/populate_atomspace.h

```diff
@@ -56,67 +59,74 @@ void populate(AtomSpace *as, const CTable &ctable);
+
+template<typename... HandleArgs>
+inline void populate_frm_ctable(CTable _ctable, HandleArgs &&... handleArgs)
+{
 	std::vector<Handle> vect1{std::forward<HandleArgs>(handleArgs)...};
 	int index = 0;
 	for (Handle h: vect1) {
 		Type type = h->get_type();
 		ValueSeq _f_values;
 		if (type == PREDICATE_NODE) {
-			boost::transform(_ctable.get_input_col_data(index), back_inserter(_f_values), bool_vertex_to_value);
+			boost::transform(_ctable.get_input_col_data(index),
+							 back_inserter(_f_values), bool_vertex_to_value);
 			ValuePtr f_proto_atom(new LinkValue(_f_values));
 			h->setValue(moses::compressed_value_key, f_proto_atom);
 		} else if (type == CONCEPT_NODE) {
 			ValueSeq _f_values;
-			boost::transform(_ctable.get_input_col_data(index), back_inserter(_f_values), GET_ENUM);
+			boost::transform(_ctable.get_input_col_data(index),
+							 back_inserter(_f_values), GET_ENUM);
 			ValuePtr f_proto_atom(new LinkValue(_f_values));
 			h->setValue(moses::compressed_value_key, f_proto_atom);
 		} else if (type == SCHEMA_NODE) {
 			std::vector<double> _f_values;
-			boost::transform(_ctable.get_input_col_data(index), back_inserter(_f_values), get_contin);
+			boost::transform(_ctable.get_input_col_data(index),
+							 back_inserter(_f_values), get_contin);
 			ValuePtr f_proto_atom(new FloatValue(_f_values));
 			h->setValue(moses::compressed_value_key, f_proto_atom);
 		} else OC_ASSERT(false, "Node Type don't exist or not implemented!");
 		index++;
 	}
 }
 
-template <typename... HandleArgs>
-inline void populate_frm_table(Table _table, HandleArgs &&... handleArgs) {
+template<typename... HandleArgs>
+inline void populate_frm_table(Table _table, HandleArgs &&... handleArgs)
+{
 	std::vector<Handle> vect1{std::forward<HandleArgs>(handleArgs)...};
 	int index = 0;
 	for (Handle h: vect1) {
-	Type type = h->get_type();
-	ValueSeq _f_values;
-	if (type == PREDICATE_NODE) {
-		boost::transform(_table.itable.get_column_data(index), back_inserter(_f_values),
-		                 bool_vertex_to_value);
-		ValuePtr f_proto_atom(new LinkValue(_f_values));
-		h->setValue(moses::value_key, f_proto_atom);
-	} else if (type == CONCEPT_NODE) {
+		Type type = h->get_type();
 		ValueSeq _f_values;
-		boost::transform(_table.itable.get_column_data(index), back_inserter(_f_values), GET_ENUM);
-		ValuePtr f_proto_atom(new LinkValue(_f_values));
-		h->setValue(moses::value_key, f_proto_atom);
-	} else if (type == SCHEMA_NODE) {
-		std::vector<double> _f_values;
-		boost::transform(_table.itable.get_column_data(index), back_inserter(_f_values), get_contin);
-		ValuePtr f_proto_atom(new FloatValue(_f_values));
-		h->setValue(moses::value_key, f_proto_atom);
-	} else OC_ASSERT(false, "Node Type don't exist or not implemented!");
-	index++;
+		if (type == PREDICATE_NODE) {
+			boost::transform(_table.itable.get_column_data(index),
+							 back_inserter(_f_values), bool_vertex_to_value);
```

Comment:
Oh, misaligned too, clearly your IDE has not been configured to support smart/mixed tabs.